matlab/src/matlab/+arrow/+array/Array.m

```diff
@@ -62,8 +62,27 @@
     end
 
     methods (Access=protected)
+        function header = getHeader(obj)
+            name = matlab.mixin.CustomDisplay.getClassNameForHeader(obj);
+            numElements = obj.NumElements;
+            % TODO: Add NumValid and NumNull as properties to Array to
```

Comment:
FYI: We can use `arrow::Array::null_count()` for them.

Reply:
Thanks for letting me know! I've created an issue for this here: #38422